public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Weather Together</title>
```

Comment:
Could the space be removed to make this `WeatherTogether`

Reply:
@Sykogst Yep! Just pushed up the change.